packages/@aws-cdk/aws-rds/README.md

```diff
@@ -45,33 +26,85 @@ By default, the master password will be generated and stored in AWS Secrets Mana
 Your cluster will be empty by default. To add a default database upon construction, specify the
 `defaultDatabaseName` attribute.
 
+### Starting an Instance Database
+To set up a instance database, define a `DatabaseInstance`. You must
+always launch a database in a VPC. Use the `vpcSubnets` attribute to control whether
+your instances will be launched privately or publicly:
+
+```ts
+const instance = new DatabaseInstance(stack, 'Instance', {
+    engine: rds.DatabaseInstanceEngine.OracleSE1,
+    instanceClass: new ec2.InstanceTypePair(ec2.InstanceClass.Burstable2, ec2.InstanceSize.Small),
+    masterUsername: 'syscdk',
+    vpc
+});
+```
+By default, the master password will be generated and stored in AWS Secrets Manager.
+
+Use `DatabaseInstanceFromSnapshot` and `DatabaseInstanceReadReplica` to create an instance from snapshot or
+a source database respectively:
+
+```ts
+new DatabaseInstanceFromSnapshot(stack, 'Instance', {
+    snapshotIdentifier: 'my-snapshot',
+    engine: rds.DatabaseInstanceEngine.Postgres,
+    instanceClass: new ec2.InstanceTypePair(ec2.InstanceClass.Burstable2, ec2.InstanceSize.Large),
+    vpc
+});
+
+new DatabaseInstanceReadReplica(stack, 'ReadReplica', {
+    sourceDatabaseInstance: sourceInstance,
+    engine: rds.DatabaseInstanceEngine.Postgres,
+    instanceClass: new ec2.InstanceTypePair(ec2.InstanceClass.Burstable2, ec2.InstanceSize.Large),
+    vpc
+});
+```
+Creating a "production" Oracle database instance with option and parameter groups:
+
+[example of setting up a production oracle instance](test/integ.instance.lit.ts)
+
+
+### Instance events
+To define Amazon CloudWatch event rules for database instances, use the `onEvent`
+method:
+
+```ts
+const rule = instance.onEvent('InstanceEvent');
```

Comment:
Is this an actual event name?

Reply:
No it's the `id`:
```
public onEvent(id: string, target?: events.IEventRuleTarget, options?: events.EventRuleProps)
```

Only the generic `onEvent` has been implemented (there are way too many events possible for RDS instances: snapshots, failover, reboot, restart, logs, security groups...)